### DISCUSSION TITLE
Device is not joined to my server

### DISCUSSION THREAD

**Discussion by ghost** [Ylianst/MeshCentral#6464] [Issues]
Hi all,
I see an issue when trying to joining a new device on a windows system.
I'm downloading the agent to the device and running it but nothing happens.
I don't the meshagent service in the service list which might mean that the agent is not even being run
I tried to disable the Windows Defender as well but did help.
It a brand new Dell device with a new windows installation.
Any help would be appreciated. Thanks.

**Comment by si458** [comment]
If its windiws 11 24h2, you need to install WMIC before you install meshagent
https://github.com/Ylianst/MeshCentral/issues/6457

**Reply by ghost** [reply]
Thanks now it works.
Didn't think to check WMIC.
Can you just explain why WMIC is important for the Mesh?

**Reply by si458** [reply]
Meshcentral has used wmic for years, it's get information about the os and devices etc, it's a command line tool that's been included for years with Windows but Microsoft decided to deprecate the feature, then remove it completely in windows 11 24h2 so u have to add it back in!
We already have a few PRs open/ready to work around using wmic just haven't had chance to test/new build of agents etc
